bootstrap/amazon_k8s_debian_stretch_master.sh

```diff
@@ -0,0 +1,75 @@
+# ------------------------------------------------------------------------------------------------------------------------
+# We are explicitly not using a templating language to inject the values as to encourage the user to limit their
+# use of templating logic in these files. By design all injected values should be able to be set at runtime,
+# and the shell script real work. If you need conditional logic, write it in bash or make another shell script.
+# ------------------------------------------------------------------------------------------------------------------------
+
+# Specify the Kubernetes version to use
+KUBERNETES_VERSION="1.8.7"
+
+apt-get update -y
+apt-get install -y \
+    apt-transport-https \
+    ca-certificates \
+    curl \
+    software-properties-common \
+    jq
+
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add -
+
+add-apt-repository \
+   "deb https://download.docker.com/linux/$(. /etc/os-release; echo "$ID") \
+   $(lsb_release -cs) \
+   stable"
+
+apt-get update && apt-get install -y docker-ce=$(apt-cache madison docker-ce | grep 17.03 | head -1 | awk '{print $3}')
+apt-get update && apt-get install -y apt-transport-https
+
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+
+cat << EOF > "/etc/apt/sources.list.d/kubernetes.list"
+deb http://apt.kubernetes.io/ kubernetes-xenial main
+EOF
+
+systemctl enable docker
+systemctl start docker
+
+apt-get update -y
+apt-get install -y \
```

Comment:
I'm not sure what recommended packages this will fetch, but other bootstrap scripts seem to install `socat`, `etables` and `cloud-utils` as well. If you can, check do we need them.

Reply:
hm seems to work with `socat`, `cloud-utils`. i'm not completely sure on whether they are actually needed.